src/components/Combobox/components/ComboboxOption/ComboboxOption.jsx

```diff
@@ -90,10 +91,16 @@ const ComboboxOption = ({
     },
     [onOptionClick, index, option]
   );
+  let tooltipContent;
+  if (disabled) {
```

Comment:
please change the logic to the following: if there is a tooltip content display it as tooltip, else take the tooltip content from "isOptionOverflowing ? label : null"